src/karma/KarmaParser.ts

```diff
@@ -29,6 +30,10 @@ export class KarmaParser {
      * at least two long. The amount to increment/decrement is equal to the number of incrementers/decrementers
      * minus one. That way `++` results in a single incrementation.
      *
+     * We also want to check if the word is a user reference and if so format it accordingly. Slack makes it
+     * easy for us, as it automatically changes the user display name to the user id before passing it ot the bot.
```

Comment:
More nitpicking, but while we're at it, there's a typo here. "passing it OT the bot" -> "passing it TO the bot"